sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -55,6 +55,24 @@ public partial class AzureKeyCredential
         public static bool operator !=(Azure.HttpRange left, Azure.HttpRange right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class JsonPatchDocument
+    {
+        public JsonPatchDocument() { }
+        public JsonPatchDocument(Azure.Core.Serialization.ObjectSerializer serializer) { }
+        public JsonPatchDocument(System.ReadOnlyMemory<byte> rawDocument) { }
```

Comment:
@alzimmermsft FYI, we've added a way to pass an existing document as `JsonPatchDocument` with no overhead.